src/types/therapist.ts

```diff
@@ -11,4 +11,12 @@ interface Imeeting {
   endDate: string,
 }
 
-export { TherapistWithUserOptional, Imeeting };
+interface Appointment {
+  id: number;

```

Comment:
when I removed the id, a database error occurred because of duplicate appointment pk, I will fix it tomorrow